server/memphis_handlers_schemas.go

```diff
@@ -334,28 +431,34 @@ func (sh SchemasHandler) RemoveSchema(c *gin.Context) {
 	if !ok {
 		return
 	}
-	schemaName := strings.ToLower(body.SchemaName)
-	exist, schema, err := IsSchemaExist(schemaName)
-	if err != nil {
-		serv.Errorf("RemoveSchema error: " + err.Error())
-		c.AbortWithStatusJSON(500, gin.H{"message": "Server error"})
-		return
-	}
-	if !exist {
-		serv.Warnf("Schema does not exist")
-		c.AbortWithStatusJSON(configuration.SHOWABLE_ERROR_STATUS_CODE, gin.H{"message": "Schema does not exist"})
-		return
+	for _, name := range body.SchemasName {
+		schemaName := strings.ToLower(name)
+		exist, schema, err := IsSchemaExist(schemaName)
```

Comment:
this is not needed, the user wants to remove schemas in case it is already not exist this is good so just insert the deleteTags into an if exist statement